It looks like we previously used a compiler/c++ lib that auto-exported
uint32_t to the scope. As this is no longer the case, export it manually.

upstream: No.

diff --git a/retroshare-gui/src/gui/statistics/Histogram.h b/retroshare-gui/src/gui/statistics/Histogram.h
index 3931b15db..1e07c701b 100644
--- a/retroshare-gui/src/gui/statistics/Histogram.h
+++ b/retroshare-gui/src/gui/statistics/Histogram.h
@@ -22,6 +22,9 @@
 #include <iostream>
 #include <cstdint>
 
+// ensure that uint32_t is available in the scope
+using std::uint32_t;
+
 class QPainter;
 
 class Histogram
